### DISCUSSION TITLE
February 3

### DISCUSSION THREAD

**Discussion by roberto-morado** [careerJumpstart/TestimonialFree#26] [Planning]
Content regarding today's meeting

**Comment by ClaraLopezUribe** [comment]
Murugueshwari and I are unclear about how/when we can contribute, especially as it relates to the front-end.  We need to clarify who is contributing in other ways (UI design, wire-frames, etc) and if/when those portions have already been discussed and/generated. We also need to better understand how the frameworks work together (back-end with front-end) to not duplicate work.

Action Items

* Murugeshwari will ask Vinod how to proceed in Zoom meeting later today and report back
* Clara will study the GitHub workflow to understand the big picture organization of information and communication
* We both will begin to study the front-end framework and how it will interact with the back-end framework

**Reply by roberto-morado** [reply]
This is great feedback!

**Comment by Saimurugeshwari** [comment]
In our meeting, Vinod and I talked about how and where we may contribute specifically (UI design, wireframes, etc.).

we need to establish a branch and set up our local goods is discussed in the reference video at (https://www.youtube.com/watch?v= RkJ230P8gY).

2. We can begin working on a minor component after setting up our local environment ( example. Login page, homepage)
3. After finishing that, we may submit a merge request.
4. We believe we can begin this along with studying the front-end framework because Clara and I are both familiar with front-end knowledge (HTML, CSS, and Javascript).
5. We can submit a merge request once each task has been accomplished.
6. Workflow, wireframes, and UI design Vinod is attempting to do that, and he plans to compete on Sunday.

**Reply by roberto-morado** [reply]
This is great feedback!
